org-docs/CHARTER.md

```diff
@@ -17,39 +17,42 @@ We refer to *data directly consumable by analyses* as "*analysis-grade* data" by
 * **minimally preprocessed**, but are
 * **safe to *consume*** directly.
 
-## 2. Steering Committee
+## 2. NiPreps Steering Committee (NSC)
 
-**2.1 Purpose**. The Steering Committee will be responsible for all technical oversight, project approval and oversight, policy oversight, and trademark management for the Organization.
-The Steering Committee is in responsible for organizing and properly promote participation in community meetings called *NiPreps Roundups*.
-The frequency, date, time and platform of these *Roundups* is determined by the Steering Committee, but shall occur at least once a calendar year.
+**2.1 Purpose**. The NSC will be responsible for all technical oversight, project approval and oversight, policy oversight, and trademark management for the Organization.
+The NSC is in responsible for organizing and properly promote participation in community meetings called *NiPreps Roundups*.
+The frequency, date, time and platform of these *Roundups* is determined by the NSC, but shall occur at least once a calendar year.
+Finally, the NSC is also ultimately responsible for securing the coordination of the *NiPreps Technical Monitoring meetings* described in the `PROJECT-CHARTER.md` document.
 
-**2.2 Composition**. The Steering Committee voting members are listed in the `STEERING-COMMITTEE.md` file in the repository.
-Voting members may be added or removed by no less than 3/4 affirmative vote of the Steering Committee.
-The number of members of the Steering Committee must always be at least three and odd.
+**2.2 Composition**. The NSC voting members are listed in the `STEERING-COMMITTEE.md` file in the repository.
+Voting members may be added or removed by no less than three-quarters (3/4) affirmative vote of the NSC.
+The number of members of the NSC is must be an odd number, and it is three (3) as of the approval of this document.
```

Comment:
Are we sure it will be three for the approval?

Reply:
No, we don't. This will need to be updated when passing this first version of the governance, if necessary.